marlowe-playground-client/src/Marlowe/Extended.purs

```diff
@@ -12,14 +12,53 @@ import Foreign.Class (class Encode, class Decode, encode, decode)
 import Foreign.Index (hasProperty)
 import Marlowe.Semantics (decodeProp)
 import Marlowe.Semantics as S
-import Text.Pretty (class Args, class Pretty, genericHasArgs, genericHasNestedArgs, genericPretty)
+import Text.Pretty (class Args, class Pretty, genericHasArgs, genericHasNestedArgs, genericPretty, pretty)
 
 class ToCore a b where
   toCore :: a -> Maybe b
 
+data ExtendedTimeout
```

Comment:
Could you add some comments here?

Reply:
why `ExtendedTimeout` and not just `Timeout`?

Reply:
See below